pylint_odoo/checkers/modules_odoo.py

```diff
@@ -149,6 +161,24 @@ def _check_duplicate_id_csv(self):
             return False
         return True
 
+    def _check_tabs_instead_of_spaces(self):
+        """Check tabs character instead of four spaces.
+        :return: False if exists errors and
+                 add list of errors in self.msg_args
+        """
+        self.msg_args = []
+        for type_file in self.config.extfiles_to_lint:
+            for ext_file in self.filter_files_ext(type_file, relpath=False):
```

Comment:
If is a js from lib folder we need jump it (Don't check it)
Because generally are external libraries that we don't wish change it